### ISSUE TITLE
[Linux] Warp doesn't work on `v0.2024.04.16.08.02.stable_01`

### ISSUE THREAD

**Issue by alokedesai** [warpdotdev/Warp#4810]
### Describe the bug
Multiple users have reported problems after updating to `v0.2024.04.16.08.02.stable_01`. 

We are actively investigating fixes for this, if you experiencing issues with this release please comment in this thread with the exact issue you're experiencing so we can debug further.

If you would like to download the prior release to unblock using Warp, you can do so here:  https://releases.warp.dev/stable/v0.2024.04.16.08.02.stable_00/warp-terminal_0.2024.04.16.08.02.stable.00_amd64.deb

-----
Moving this over from https://github.com/warpdotdev/Warp/issues/4807.

@meatballs can't start Warp after the latest update. Upon running `warp-terminal`, no window opens.

### To reproduce

1. Run `warp-terminal`
2. Notice that no window opens.

### Operating system

Linux

### Operating system and version

Ubuntu 22.04

### Regression

Yes, this bug started recently or with an X Warp version

### Does this block you from using Warp daily?

No

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by alokedesai** [comment]
@meatballs What's unclear to me is why the discrete GPU does not appear in your Warp logs.

Two things that I think would be helpful so we can further debug this:

1) Install `mesa-utils` (which includes `eglinfo`)  and rerun `warp-terminal --dump-debug-info` (The output of `eglinfo` should help further guide us here)
2) Run the prior version of Warp (available at https://releases.warp.dev/stable/v0.2024.04.23.08.01.stable_00/warp-terminal_0.2024.04.16.08.02.stable.00_amd64.deb) and include the output of Warp logs so we can understand the difference between the two releases that could have caused this problem.

**Comment by meatballs** [comment]
current release after installing mesa-utils:
```
Warp version: Some("v0.2024.04.16.08.02.stable_01")
uname(1) output: Linux <redacted> 6.5.0-28-generic #29~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Apr  4 14:39:20 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
Package type: PackageManager(Apt)
Windowing system: X11 { is_x_wayland: false }
Window manager name: i3
##################################################
# wgpu Adapters
##################################################
Cpu: llvmpipe (LLVM 15.0.7, 256 bits)
	Backend: Vulkan
	Driver: llvmpipe (Mesa 23.2.1-1ubuntu3.1~22.04.2 (LLVM 15.0.7))
Cpu: llvmpipe (LLVM 15.0.7, 256 bits)
	Backend: Gl
	Driver: ?
##################################################
# lspci(8) output
##################################################
00:00.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Root Complex
00:00.2 IOMMU: Advanced Micro Devices, Inc. [AMD] Starship/Matisse IOMMU
00:01.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:01.1 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse GPP Bridge
00:01.3 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse GPP Bridge
00:02.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:03.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:04.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:05.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:07.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:07.1 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Internal PCIe GPP Bridge 0 to bus[E:B]
00:08.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:08.1 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Internal PCIe GPP Bridge 0 to bus[E:B]
00:08.2 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Internal PCIe GPP Bridge 0 to bus[E:B]
00:08.3 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Internal PCIe GPP Bridge 0 to bus[E:B]
00:14.0 SMBus: Advanced Micro Devices, Inc. [AMD] FCH SMBus Controller (rev 61)
00:14.3 ISA bridge: Advanced Micro Devices, Inc. [AMD] FCH LPC Bridge (rev 51)
00:18.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 0
00:18.1 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 1
00:18.2 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 2
00:18.3 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 3
00:18.4 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 4
00:18.5 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 5
00:18.6 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 6
00:18.7 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 7
01:00.0 Non-Volatile memory controller: Phison Electronics Corporation E12 NVMe Controller (rev 01)
03:00.0 USB controller: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset USB 3.1 XHCI Controller (rev 01)
03:00.1 SATA controller: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset SATA Controller (rev 01)
03:00.2 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Bridge (rev 01)
20:00.0 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Port (rev 01)
20:01.0 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Port (rev 01)
20:04.0 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Port (rev 01)
20:06.0 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Port (rev 01)
20:07.0 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Port (rev 01)
22:00.0 Ethernet controller: Realtek Semiconductor Co., Ltd. RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller (rev 15)
25:00.0 VGA compatible controller: NVIDIA Corporation TU104 [GeForce RTX 2070 SUPER] (rev a1)
25:00.1 Audio device: NVIDIA Corporation TU104 HD Audio Controller (rev a1)
25:00.2 USB controller: NVIDIA Corporation TU104 USB 3.1 Host Controller (rev a1)
25:00.3 Serial bus controller: NVIDIA Corporation TU104 USB Type-C UCSI Controller (rev a1)
28:00.0 Network controller: Intel Corporation Wireless-AC 9260 (rev 29)
29:00.0 Non-Essential Instrumentation [1300]: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Function
2a:00.0 Non-Essential Instrumentation [1300]: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Reserved SPP
2a:00.1 Encryption controller: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Cryptographic Coprocessor PSPCPP
2a:00.3 USB controller: Advanced Micro Devices, Inc. [AMD] Matisse USB 3.0 Host Controller
2a:00.4 Audio device: Advanced Micro Devices, Inc. [AMD] Starship/Matisse HD Audio Controller
30:00.0 SATA controller: Advanced Micro Devices, Inc. [AMD] FCH SATA Controller [AHCI mode] (rev 51)
31:00.0 SATA controller: Advanced Micro Devices, Inc. [AMD] FCH SATA Controller [AHCI mode] (rev 51)

##################################################
# vulkaninfo(1) output
##################################################
sh: 1: vulkaninfo: not found

##################################################
# eglinfo(1) output
##################################################
EGL client extensions string:
    EGL_EXT_device_base EGL_EXT_device_enumeration EGL_EXT_device_query
    EGL_EXT_platform_base EGL_KHR_client_get_all_proc_addresses
    EGL_EXT_client_extensions EGL_KHR_debug EGL_EXT_platform_device
    EGL_EXT_platform_wayland EGL_KHR_platform_wayland
    EGL_EXT_platform_x11 EGL_KHR_platform_x11 EGL_EXT_platform_xcb
    EGL_MESA_platform_gbm EGL_KHR_platform_gbm
    EGL_MESA_platform_surfaceless

GBM platform:
EGL API version: 1.5
EGL vendor string: Mesa Project
EGL version string: 1.5
EGL client APIs: OpenGL OpenGL_ES 
EGL extensions string:
    EGL_ANDROID_blob_cache EGL_EXT_buffer_age
    EGL_EXT_create_context_robustness EGL_EXT_image_dma_buf_import
    EGL_EXT_image_dma_buf_import_modifiers EGL_KHR_cl_event2
    EGL_KHR_config_attribs EGL_KHR_context_flush_control
    EGL_KHR_create_context EGL_KHR_create_context_no_error
    EGL_KHR_fence_sync EGL_KHR_get_all_proc_addresses
    EGL_KHR_gl_colorspace EGL_KHR_gl_renderbuffer_image
    EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_3D_image
    EGL_KHR_gl_texture_cubemap_image EGL_KHR_image EGL_KHR_image_base
    EGL_KHR_image_pixmap EGL_KHR_no_config_context EGL_KHR_reusable_sync
    EGL_KHR_surfaceless_context EGL_EXT_pixel_format_float
    EGL_KHR_wait_sync EGL_MESA_configless_context
    EGL_MESA_image_dma_buf_export EGL_MESA_query_driver
Configurations:
     bf lv colorbuffer dp st  ms    vis   cav bi  renderable  supported
  id sz  l  r  g  b  a th cl ns b    id   eat nd gl es es2 vg surfaces 
---------------------------------------------------------------------
0x01 32  0 10 10 10  2  0  0  0 0 0x30335241--         y  y  y     win
0x02 32  0 10 10 10  2 16  0  0 0 0x30335241--         y  y  y     win
0x03 32  0 10 10 10  2 24  0  0 0 0x30335241--         y  y  y     win
0x04 32  0 10 10 10  2 24  8  0 0 0x30335241--         y  y  y     win
0x05 32  0 10 10 10  2 32  0  0 0 0x30335241--         y  y  y     win
0x06 32  0 10 10 10  2  0  0  4 1 0x30335241--         y  y  y     win
0x07 32  0 10 10 10  2 16  0  4 1 0x30335241--         y  y  y     win
0x08 32  0 10 10 10  2 24  0  4 1 0x30335241--         y  y  y     win
0x09 32  0 10 10 10  2 24  8  4 1 0x30335241--         y  y  y     win
0x0a 32  0 10 10 10  2 32  0  4 1 0x30335241--         y  y  y     win
0x0b 30  0 10 10 10  0  0  0  0 0 0x30335258--         y  y  y     win
0x0c 30  0 10 10 10  0 16  0  0 0 0x30335258--         y  y  y     win
0x0d 30  0 10 10 10  0 24  0  0 0 0x30335258--         y  y  y     win
0x0e 30  0 10 10 10  0 24  8  0 0 0x30335258--         y  y  y     win
0x0f 30  0 10 10 10  0 32  0  0 0 0x30335258--         y  y  y     win
0x10 30  0 10 10 10  0  0  0  4 1 0x30335258--         y  y  y     win
0x11 30  0 10 10 10  0 16  0  4 1 0x30335258--         y  y  y     win
0x12 30  0 10 10 10  0 24  0  4 1 0x30335258--         y  y  y     win
0x13 30  0 10 10 10  0 24  8  4 1 0x30335258--         y  y  y     win
0x14 30  0 10 10 10  0 32  0  4 1 0x30335258--         y  y  y     win
0x15 32  0  8  8  8  8  0  0  0 0 0x34325241--         y  y  y     win
0x16 32  0  8  8  8  8 16  0  0 0 0x34325241--         y  y  y     win
0x17 32  0  8  8  8  8 24  0  0 0 0x34325241--         y  y  y     win
0x18 32  0  8  8  8  8 24  8  0 0 0x34325241--         y  y  y     win
0x19 32  0  8  8  8  8 32  0  0 0 0x34325241--         y  y  y     win
0x1a 32  0  8  8  8  8  0  0  4 1 0x34325241--         y  y  y     win
0x1b 32  0  8  8  8  8 16  0  4 1 0x34325241--         y  y  y     win
0x1c 32  0  8  8  8  8 24  0  4 1 0x34325241--         y  y  y     win
0x1d 32  0  8  8  8  8 24  8  4 1 0x34325241--         y  y  y     win
0x1e 32  0  8  8  8  8 32  0  4 1 0x34325241--         y  y  y     win
0x1f 24  0  8  8  8  0  0  0  0 0 0x34325258--         y  y  y     win
0x20 24  0  8  8  8  0 16  0  0 0 0x34325258--         y  y  y     win
0x21 24  0  8  8  8  0 24  0  0 0 0x34325258--         y  y  y     win
0x22 24  0  8  8  8  0 24  8  0 0 0x34325258--         y  y  y     win
0x23 24  0  8  8  8  0 32  0  0 0 0x34325258--         y  y  y     win
0x24 24  0  8  8  8  0  0  libEGL warning: DRI2: failed to authenticate
0  4 1 0x34325258--         y  y  y     win
0x25 24  0  8  8  8  0 16  0  4 1 0x34325258--         y  y  y     win
0x26 24  0  8  8  8  0 24  0  4 1 0x34325258--         y  y  y     win
0x27 24  0  8  8  8  0 24  8  4 1 0x34325258--         y  y  y     win
0x28 24  0  8  8  8  0 32  0  4 1 0x34325258--         y  y  y     win
0x29 16  0  5  6  5  0  0  0  0 0 0x36314752--         y  y  y     win
0x2a 16  0  5  6  5  0 16  0  0 0 0x36314752--         y  y  y     win
0x2b 16  0  5  6  5  0 24  0  0 0 0x36314752--         y  y  y     win
0x2c 16  0  5  6  5  0 24  8  0 0 0x36314752--         y  y  y     win
0x2d 16  0  5  6  5  0 32  0  0 0 0x36314752--         y  y  y     win
0x2e 16  0  5  6  5  0  0  0  4 1 0x36314752--         y  y  y     win
0x2f 16  0  5  6  5  0 16  0  4 1 0x36314752--         y  y  y     win
0x30 16  0  5  6  5  0 24  0  4 1 0x36314752--         y  y  y     win
0x31 16  0  5  6  5  0 24  8  4 1 0x36314752--         y  y  y     win
0x32 16  0  5  6  5  0 32  0  4 1 0x36314752--         y  y  y     win
0x33 64  0 16 16 16 16  0  0  0 0 0x48344241--         y  y  y     win
0x34 64  0 16 16 16 16 16  0  0 0 0x48344241--         y  y  y     win
0x35 64  0 16 16 16 16 24  0  0 0 0x48344241--         y  y  y     win
0x36 64  0 16 16 16 16 24  8  0 0 0x48344241--         y  y  y     win
0x37 64  0 16 16 16 16 32  0  0 0 0x48344241--         y  y  y     win
0x38 64  0 16 16 16 16  0  0  4 1 0x48344241--         y  y  y     win
0x39 64  0 16 16 16 16 16  0  4 1 0x48344241--         y  y  y     win
0x3a 64  0 16 16 16 16 24  0  4 1 0x48344241--         y  y  y     win
0x3b 64  0 16 16 16 16 24  8  4 1 0x48344241--         y  y  y     win
0x3c 64  0 16 16 16 16 32  0  4 1 0x48344241--         y  y  y     win
0x3d 48  0 16 16 16  0  0  0  0 0 0x48344258--         y  y  y     win
0x3e 48  0 16 16 16  0 16  0  0 0 0x48344258--         y  y  y     win
0x3f 48  0 16 16 16  0 24  0  0 0 0x48344258--         y  y  y     win
0x40 48  0 16 16 16  0 24  8  0 0 0x48344258--         y  y  y     win
0x41 48  0 16 16 16  0 32  0  0 0 0x48344258--         y  y  y     win
0x42 48  0 16 16 16  0  0  0  4 1 0x48344258--         y  y  y     win
0x43 48  0 16 16 16  0 16  0  4 1 0x48344258--         y  y  y     win
0x44 48  0 16 16 16  0 24  0  4 1 0x48344258--         y  y  y     win
0x45 48  0 16 16 16  0 24  8  4 1 0x48344258--         y  y  y     win
0x46 48  0 16 16 16  0 32  0  4 1 0x48344258--         y  y  y     win

Wayland platform:
eglinfo: eglInitialize failed

X11 platform:
EGL API version: 1.5
EGL vendor string: Mesa Project
EGL version string: 1.5
EGL client APIs: OpenGL OpenGL_ES 
EGL extensions string:
    EGL_ANGLE_sync_control_rate EGL_EXT_create_context_robustness
    EGL_KHR_cl_event2 EGL_KHR_config_attribs
    EGL_KHR_context_flush_control EGL_KHR_create_context
    EGL_KHR_create_context_no_error EGL_KHR_fence_sync
    EGL_KHR_get_all_proc_addresses EGL_KHR_gl_colorspace
    EGL_KHR_gl_renderbuffer_image EGL_KHR_gl_texture_2D_image
    EGL_KHR_gl_texture_3D_image EGL_KHR_gl_texture_cubemap_image
    EGL_KHR_image_base EGL_KHR_no_config_context EGL_KHR_reusable_sync
    EGL_KHR_surfaceless_context EGL_EXT_pixel_format_float
    EGL_KHR_wait_sync EGL_MESA_configless_context EGL_MESA_drm_image
    EGL_MESA_query_driver
Configurations:
     bf lv colorbuffer dp st  ms    vis   cav bi  renderable  supported
  id sz  l  r  g  b  a th cl ns b    id   eat nd gl es es2 vg surfaces 
---------------------------------------------------------------------
0x01 32  0  8  8  8  8  0  0  0 0 0x21TC      a  y  y  y     win,pb,pix
0x02 32  0  8  8  8  8 16  0  0 0 0x21TC      a  y  y  y     win,pb,pix
0x03 32  0  8  8  8  8 24  0  0 0 0x21TC      a  y  y  y     win,pb,pix
0x04 32  0  8  8  8  8 24  8  0 0 0x21TC      a  y  y  y     win,pb,pix
0x05 32  0  8  8  8  8 32  0  0 0 0x21TC      a  y  y  y     win,pb,pix
0x06 32  0  8  8  8  8  0  0  4 1 0x21TC      a  y  y  y     win,pb,pix
0x07 32  0  8  8  8  8 16  0  4 1 0x21TC      a  y  y  y     win,pb,pix
0x08 32  0  8  8  8  8 24  0  4 1 0x21TC      a  y  y  y     win,pb,pix
0x09 32  0  8  8  8  8 24  8  4 1 0x21TC      a  y  y  y     win,pb,pix
0x0a 32  0  8  8  8  8 32  0  4 1 0x21TC      a  y  y  y     win,pb,pix
0x0b 24  0  8  8  8  0  0  0  0 0 0x21TC      y  y  y  y     win,pb,pix
0x0c 24  0  8  8  8  0 16  0  0 0 0x21TC      y  y  y  y     win,pb,pix
0x0d 24  0  8  8  8  0 24  0  0 0 0x21TC      y  y  y  y     win,pb,pix
0x0e 24  0  8  8  8  0 24  8  0 0 0x21TC      y  y  y  y     win,pb,pix
0x0f 24  0  8  8  8  0 32  0  0 0 0x21TC      y  y  y  y     win,pb,pix
0x10 24  0  8  8  8  0  0  0  4 1 0x21TC      y  y  y  y     win,pb,pix
0x11 24  0  8  8  8  0 16  0  4 1 0x21TC      y  y  y  y     win,pb,pix
0x12 24  0  8  8  8  0 24  0  4 1 0x21TC      y  y  y  y     win,pb,pix
0x13 24  0  8  8  8  0 24  8  4 1 0x21TC      y  y  y  y     win,pb,pix
0x14 24  0  8  8  8  0 32  0  4 1 0x21TC      y  y  y  y     win,pb,pix
0x15 32  0  8  8  8  8  0  0  0 0 0x22DC      a  y  y  y     win,pb,pix
0x16 32  0  8  8  8  8 16  0  0 0 0x22DC      a  y  y  y     win,pb,pix
0x17 32  0  8  8  8  8 24  0  0 0 0x22DC      a  y  y  y     win,pb,pix
0x18 32  0  8  8  8  8 24  8  0 0 0x22DC      a  y  y  y     win,pb,pix
0x19 32  0  8  8  8  8 32  0  0 0 0x22DC      a  y  y  y     win,pb,pix
0x1a 32  0  8  8  8  8  0  0  4 1 0x22DC      a  y  y  y     win,pb,pix
0x1b 32  0  8  8  8  8 16  0  4 1 0x22DC      a  y  y  y     win,pb,pix
0x1c 32  0  8  8  8  8 24  0  4 1 0x22DC      a  y  y  y     win,pb,pix
0x1d 32  0  8  8  8  8 24  8  4 1 0x22DC      a  y  y  y     win,pb,pix
0x1e 32  0  8  8  8  8 32  0  4 1 0x22DC      a  y  y  y     win,pb,pix
0x1f 24  0  8  8  8  0  0  0  0 0 0x22DC      y  y  y  y     win,pb,pix
0x20 24  0  8  8  8  0 16  0  0 0 0x22DC      y  y  y  y     win,pb,pix
0x21 24  0  8  8  8  0 24  0  0 0 0x22DC      y  y  y  y     win,pb,pix
0x22 24  0  8  8  8  0 24  8  0 0 0x22DC      y  y  y  y     win,pb,pix
0x23 24  0  8  8  8  0 32  0  0 0 0x22DC      y  y  y  y     win,pb,pix
0x24 24  0  8  8  8  0  0  0  4 1 0x22DC      y  y  y  y     win,pb,pix
0x25 24  0  8  8  8  0 16  0  4 1 0x22DC      y  y  y  y     win,pb,pix
0x26 24  0  8  8  8  0 24  0  4 1 0x22DC      y  y  y  y     win,pb,pix
0x27 24  0  8  8  8  0 24  8  4 1 0x22DC      y  y  y  y     win,pb,pix
0x28 24  0  8  8  8  0 32  0  4 1 0x22DC      y  y  y  y     win,pb,pix

Device platform:
eglinfo: eglInitialize failed
```

**Comment by meatballs** [comment]
There's nothing available to download at the link for the previous release.

**Comment by alokedesai** [comment]
@meatballs Updated the link so it should now work, thanks for catching!

**Comment by meatballs** [comment]
previous version working fine:
```
warp-terminal --dump-debug-info
Warp version: Some("v0.2024.04.16.08.02.stable_00")
uname(1) output: Linux <redacted> 6.5.0-28-generic #29~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Apr  4 14:39:20 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
Package type: PackageManager(Apt)
Windowing system: X11 { is_x_wayland: false }
Window manager name: i3
##################################################
# wgpu Adapters
##################################################
libEGL warning: DRI2: failed to authenticate
Cpu: llvmpipe (LLVM 15.0.7, 256 bits)
	Backend: Vulkan
	Driver: llvmpipe (Mesa 23.2.1-1ubuntu3.1~22.04.2 (LLVM 15.0.7))
Cpu: llvmpipe (LLVM 15.0.7, 256 bits)
	Backend: Gl
	Driver: ?
##################################################
# lspci(8) output
##################################################
00:00.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Root Complex
00:00.2 IOMMU: Advanced Micro Devices, Inc. [AMD] Starship/Matisse IOMMU
00:01.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:01.1 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse GPP Bridge
00:01.3 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse GPP Bridge
00:02.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:03.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:04.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:05.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:07.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:07.1 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Internal PCIe GPP Bridge 0 to bus[E:B]
00:08.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Host Bridge
00:08.1 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Internal PCIe GPP Bridge 0 to bus[E:B]
00:08.2 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Internal PCIe GPP Bridge 0 to bus[E:B]
00:08.3 PCI bridge: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Internal PCIe GPP Bridge 0 to bus[E:B]
00:14.0 SMBus: Advanced Micro Devices, Inc. [AMD] FCH SMBus Controller (rev 61)
00:14.3 ISA bridge: Advanced Micro Devices, Inc. [AMD] FCH LPC Bridge (rev 51)
00:18.0 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 0
00:18.1 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 1
00:18.2 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 2
00:18.3 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 3
00:18.4 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 4
00:18.5 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 5
00:18.6 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 6
00:18.7 Host bridge: Advanced Micro Devices, Inc. [AMD] Matisse/Vermeer Data Fabric: Device 18h; Function 7
01:00.0 Non-Volatile memory controller: Phison Electronics Corporation E12 NVMe Controller (rev 01)
03:00.0 USB controller: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset USB 3.1 XHCI Controller (rev 01)
03:00.1 SATA controller: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset SATA Controller (rev 01)
03:00.2 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Bridge (rev 01)
20:00.0 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Port (rev 01)
20:01.0 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Port (rev 01)
20:04.0 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Port (rev 01)
20:06.0 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Port (rev 01)
20:07.0 PCI bridge: Advanced Micro Devices, Inc. [AMD] 400 Series Chipset PCIe Port (rev 01)
22:00.0 Ethernet controller: Realtek Semiconductor Co., Ltd. RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller (rev 15)
25:00.0 VGA compatible controller: NVIDIA Corporation TU104 [GeForce RTX 2070 SUPER] (rev a1)
25:00.1 Audio device: NVIDIA Corporation TU104 HD Audio Controller (rev a1)
25:00.2 USB controller: NVIDIA Corporation TU104 USB 3.1 Host Controller (rev a1)
25:00.3 Serial bus controller: NVIDIA Corporation TU104 USB Type-C UCSI Controller (rev a1)
28:00.0 Network controller: Intel Corporation Wireless-AC 9260 (rev 29)
29:00.0 Non-Essential Instrumentation [1300]: Advanced Micro Devices, Inc. [AMD] Starship/Matisse PCIe Dummy Function
2a:00.0 Non-Essential Instrumentation [1300]: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Reserved SPP
2a:00.1 Encryption controller: Advanced Micro Devices, Inc. [AMD] Starship/Matisse Cryptographic Coprocessor PSPCPP
2a:00.3 USB controller: Advanced Micro Devices, Inc. [AMD] Matisse USB 3.0 Host Controller
2a:00.4 Audio device: Advanced Micro Devices, Inc. [AMD] Starship/Matisse HD Audio Controller
30:00.0 SATA controller: Advanced Micro Devices, Inc. [AMD] FCH SATA Controller [AHCI mode] (rev 51)
31:00.0 SATA controller: Advanced Micro Devices, Inc. [AMD] FCH SATA Controller [AHCI mode] (rev 51)

##################################################
# vulkaninfo(1) output
##################################################
sh: 1: vulkaninfo: not found

##################################################
# eglinfo(1) output
##################################################
EGL client extensions string:
    EGL_EXT_device_base EGL_EXT_device_enumeration EGL_EXT_device_query
    EGL_EXT_platform_base EGL_KHR_client_get_all_proc_addresses
    EGL_EXT_client_extensions EGL_KHR_debug EGL_EXT_platform_device
    EGL_EXT_platform_wayland EGL_KHR_platform_wayland
    EGL_EXT_platform_x11 EGL_KHR_platform_x11 EGL_EXT_platform_xcb
    EGL_MESA_platform_gbm EGL_KHR_platform_gbm
    EGL_MESA_platform_surfaceless

GBM platform:
EGL API version: 1.5
EGL vendor string: Mesa Project
EGL version string: 1.5
EGL client APIs: OpenGL OpenGL_ES 
EGL extensions string:
    EGL_ANDROID_blob_cache EGL_EXT_buffer_age
    EGL_EXT_create_context_robustness EGL_EXT_image_dma_buf_import
    EGL_EXT_image_dma_buf_import_modifiers EGL_KHR_cl_event2
    EGL_KHR_config_attribs EGL_KHR_context_flush_control
    EGL_KHR_create_context EGL_KHR_create_context_no_error
    EGL_KHR_fence_sync EGL_KHR_get_all_proc_addresses
    EGL_KHR_gl_colorspace EGL_KHR_gl_renderbuffer_image
    EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_3D_image
    EGL_KHR_gl_texture_cubemap_image EGL_KHR_image EGL_KHR_image_base
    EGL_KHR_image_pixmap EGL_KHR_no_config_context EGL_KHR_reusable_sync
    EGL_KHR_surfaceless_context EGL_EXT_pixel_format_float
    EGL_KHR_wait_sync EGL_MESA_configless_context
    EGL_MESA_image_dma_buf_export EGL_MESA_query_driver
Configurations:
     bf lv colorbuffer dp st  ms    vis   cav bi  renderable  supported
  id sz  l  r  g  b  a th cl ns b    id   eat nd gl es es2 vg surfaces 
---------------------------------------------------------------------
0x01 32  0 10 10 10  2  0  0  0 0 0x30335241--         y  y  y     win
0x02 32  0 10 10 10  2 16  0  0 0 0x30335241--         y  y  y     win
0x03 32  0 10 10 10  2 24  0  0 0 0x30335241--         y  y  y     win
0x04 32  0 10 10 10  2 24  8  0 0 0x30335241--         y  y  y     win
0x05 32  0 10 10 10  2 32  0  0 0 0x30335241--         y  y  y     win
0x06 32  0 10 10 10  2  0  0  4 1 0x30335241--         y  y  y     win
0x07 32  0 10 10 10  2 16  0  4 1 0x30335241--         y  y  y     win
0x08 32  0 10 10 10  2 24  0  4 1 0x30335241--         y  y  y     win
0x09 32  0 10 10 10  2 24  8  4 1 0x30335241--         y  y  y     win
0x0a 32  0 10 10 10  2 32  0  4 1 0x30335241--         y  y  y     win
0x0b 30  0 10 10 10  0  0  0  0 0 0x30335258--         y  y  y     win
0x0c 30  0 10 10 10  0 16  0  0 0 0x30335258--         y  y  y     win
0x0d 30  0 10 10 10  0 24  0  0 0 0x30335258--         y  y  y     win
0x0e 30  0 10 10 10  0 24  8  0 0 0x30335258--         y  y  y     win
0x0f 30  0 10 10 10  0 32  0  0 0 0x30335258--         y  y  y     win
0x10 30  0 10 10 10  0  0  0  4 1 0x30335258--         y  y  y     win
0x11 30  0 10 10 10  0 16  0  4 1 0x30335258--         y  y  y     win
0x12 30  0 10 10 10  0 24  0  4 1 0x30335258--         y  y  y     win
0x13 30  0 10 10 10  0 24  8  4 1 0x30335258--         y  y  y     win
0x14 30  0 10 10 10  0 32  0  4 1 0x30335258--         y  y  y     win
0x15 32  0  8  8  8  8  0  0  0 0 0x34325241--         y  y  y     win
0x16 32  0  8  8  8  8 16  0  0 0 0x34325241--         y  y  y     win
0x17 32  0  8  8  8  8 24  0  0 0 0x34325241--         y  y  y     win
0x18 32  0  8  8  8  8 24  8  0 0 0x34325241--         y  y  y     win
0x19 32  0  8  8  8  8 32  0  0 0 0x34325241--         y  y  y     win
0x1a 32  0  8  8  8  8  0  0  4 1 0x34325241--         y  y  y     win
0x1b 32  0  8  8  8  8 16  0  4 1 0x34325241--         y  y  y     win
0x1c 32  0  8  8  8  8 24  0  4 1 0x34325241--         y  y  y     win
0x1d 32  0  8  8  8  8 24  8  4 1 0x34325241--         y  y  y     win
0x1e 32  0  8  8  8  8 32  0  4 1 0x34325241--         y  y  y     win
0x1f 24  0  8  8  8  0  0  0  0 0 0x34325258--         y  y  y     win
0x20 24  0  8  8  8  0 16  0  0 0 0x34325258--         y  y  y     win
0x21 24  0  8  8  8  0 24  0  0 0 0x34325258--         y  y  y     win
0x22 24  0  8  8  8  0 24  8  0 0 0x34325258--         y  y  y     win
0x23 24  0  8  8  8  0 32  0  0 0 0x34325258--         y  y  y     win
0x24 24  0  8  8  8  0  0  libEGL warning: DRI2: failed to authenticate
0  4 1 0x34325258--         y  y  y     win
0x25 24  0  8  8  8  0 16  0  4 1 0x34325258--         y  y  y     win
0x26 24  0  8  8  8  0 24  0  4 1 0x34325258--         y  y  y     win
0x27 24  0  8  8  8  0 24  8  4 1 0x34325258--         y  y  y     win
0x28 24  0  8  8  8  0 32  0  4 1 0x34325258--         y  y  y     win
0x29 16  0  5  6  5  0  0  0  0 0 0x36314752--         y  y  y     win
0x2a 16  0  5  6  5  0 16  0  0 0 0x36314752--         y  y  y     win
0x2b 16  0  5  6  5  0 24  0  0 0 0x36314752--         y  y  y     win
0x2c 16  0  5  6  5  0 24  8  0 0 0x36314752--         y  y  y     win
0x2d 16  0  5  6  5  0 32  0  0 0 0x36314752--         y  y  y     win
0x2e 16  0  5  6  5  0  0  0  4 1 0x36314752--         y  y  y     win
0x2f 16  0  5  6  5  0 16  0  4 1 0x36314752--         y  y  y     win
0x30 16  0  5  6  5  0 24  0  4 1 0x36314752--         y  y  y     win
0x31 16  0  5  6  5  0 24  8  4 1 0x36314752--         y  y  y     win
0x32 16  0  5  6  5  0 32  0  4 1 0x36314752--         y  y  y     win
0x33 64  0 16 16 16 16  0  0  0 0 0x48344241--         y  y  y     win
0x34 64  0 16 16 16 16 16  0  0 0 0x48344241--         y  y  y     win
0x35 64  0 16 16 16 16 24  0  0 0 0x48344241--         y  y  y     win
0x36 64  0 16 16 16 16 24  8  0 0 0x48344241--         y  y  y     win
0x37 64  0 16 16 16 16 32  0  0 0 0x48344241--         y  y  y     win
0x38 64  0 16 16 16 16  0  0  4 1 0x48344241--         y  y  y     win
0x39 64  0 16 16 16 16 16  0  4 1 0x48344241--         y  y  y     win
0x3a 64  0 16 16 16 16 24  0  4 1 0x48344241--         y  y  y     win
0x3b 64  0 16 16 16 16 24  8  4 1 0x48344241--         y  y  y     win
0x3c 64  0 16 16 16 16 32  0  4 1 0x48344241--         y  y  y     win
0x3d 48  0 16 16 16  0  0  0  0 0 0x48344258--         y  y  y     win
0x3e 48  0 16 16 16  0 16  0  0 0 0x48344258--         y  y  y     win
0x3f 48  0 16 16 16  0 24  0  0 0 0x48344258--         y  y  y     win
0x40 48  0 16 16 16  0 24  8  0 0 0x48344258--         y  y  y     win
0x41 48  0 16 16 16  0 32  0  0 0 0x48344258--         y  y  y     win
0x42 48  0 16 16 16  0  0  0  4 1 0x48344258--         y  y  y     win
0x43 48  0 16 16 16  0 16  0  4 1 0x48344258--         y  y  y     win
0x44 48  0 16 16 16  0 24  0  4 1 0x48344258--         y  y  y     win
0x45 48  0 16 16 16  0 24  8  4 1 0x48344258--         y  y  y     win
0x46 48  0 16 16 16  0 32  0  4 1 0x48344258--         y  y  y     win

Wayland platform:
eglinfo: eglInitialize failed

X11 platform:
EGL API version: 1.5
EGL vendor string: Mesa Project
EGL version string: 1.5
EGL client APIs: OpenGL OpenGL_ES 
EGL extensions string:
    EGL_ANGLE_sync_control_rate EGL_EXT_create_context_robustness
    EGL_KHR_cl_event2 EGL_KHR_config_attribs
    EGL_KHR_context_flush_control EGL_KHR_create_context
    EGL_KHR_create_context_no_error EGL_KHR_fence_sync
    EGL_KHR_get_all_proc_addresses EGL_KHR_gl_colorspace
    EGL_KHR_gl_renderbuffer_image EGL_KHR_gl_texture_2D_image
    EGL_KHR_gl_texture_3D_image EGL_KHR_gl_texture_cubemap_image
    EGL_KHR_image_base EGL_KHR_no_config_context EGL_KHR_reusable_sync
    EGL_KHR_surfaceless_context EGL_EXT_pixel_format_float
    EGL_KHR_wait_sync EGL_MESA_configless_context EGL_MESA_drm_image
    EGL_MESA_query_driver
Configurations:
     bf lv colorbuffer dp st  ms    vis   cav bi  renderable  supported
  id sz  l  r  g  b  a th cl ns b    id   eat nd gl es es2 vg surfaces 
---------------------------------------------------------------------
0x01 32  0  8  8  8  8  0  0  0 0 0x21TC      a  y  y  y     win,pb,pix
0x02 32  0  8  8  8  8 16  0  0 0 0x21TC      a  y  y  y     win,pb,pix
0x03 32  0  8  8  8  8 24  0  0 0 0x21TC      a  y  y  y     win,pb,pix
0x04 32  0  8  8  8  8 24  8  0 0 0x21TC      a  y  y  y     win,pb,pix
0x05 32  0  8  8  8  8 32  0  0 0 0x21TC      a  y  y  y     win,pb,pix
0x06 32  0  8  8  8  8  0  0  4 1 0x21TC      a  y  y  y     win,pb,pix
0x07 32  0  8  8  8  8 16  0  4 1 0x21TC      a  y  y  y     win,pb,pix
0x08 32  0  8  8  8  8 24  0  4 1 0x21TC      a  y  y  y     win,pb,pix
0x09 32  0  8  8  8  8 24  8  4 1 0x21TC      a  y  y  y     win,pb,pix
0x0a 32  0  8  8  8  8 32  0  4 1 0x21TC      a  y  y  y     win,pb,pix
0x0b 24  0  8  8  8  0  0  0  0 0 0x21TC      y  y  y  y     win,pb,pix
0x0c 24  0  8  8  8  0 16  0  0 0 0x21TC      y  y  y  y     win,pb,pix
0x0d 24  0  8  8  8  0 24  0  0 0 0x21TC      y  y  y  y     win,pb,pix
0x0e 24  0  8  8  8  0 24  8  0 0 0x21TC      y  y  y  y     win,pb,pix
0x0f 24  0  8  8  8  0 32  0  0 0 0x21TC      y  y  y  y     win,pb,pix
0x10 24  0  8  8  8  0  0  0  4 1 0x21TC      y  y  y  y     win,pb,pix
0x11 24  0  8  8  8  0 16  0  4 1 0x21TC      y  y  y  y     win,pb,pix
0x12 24  0  8  8  8  0 24  0  4 1 0x21TC      y  y  y  y     win,pb,pix
0x13 24  0  8  8  8  0 24  8  4 1 0x21TC      y  y  y  y     win,pb,pix
0x14 24  0  8  8  8  0 32  0  4 1 0x21TC      y  y  y  y     win,pb,pix
0x15 32  0  8  8  8  8  0  0  0 0 0x22DC      a  y  y  y     win,pb,pix
0x16 32  0  8  8  8  8 16  0  0 0 0x22DC      a  y  y  y     win,pb,pix
0x17 32  0  8  8  8  8 24  0  0 0 0x22DC      a  y  y  y     win,pb,pix
0x18 32  0  8  8  8  8 24  8  0 0 0x22DC      a  y  y  y     win,pb,pix
0x19 32  0  8  8  8  8 32  0  0 0 0x22DC      a  y  y  y     win,pb,pix
0x1a 32  0  8  8  8  8  0  0  4 1 0x22DC      a  y  y  y     win,pb,pix
0x1b 32  0  8  8  8  8 16  0  4 1 0x22DC      a  y  y  y     win,pb,pix
0x1c 32  0  8  8  8  8 24  0  4 1 0x22DC      a  y  y  y     win,pb,pix
0x1d 32  0  8  8  8  8 24  8  4 1 0x22DC      a  y  y  y     win,pb,pix
0x1e 32  0  8  8  8  8 32  0  4 1 0x22DC      a  y  y  y     win,pb,pix
0x1f 24  0  8  8  8  0  0  0  0 0 0x22DC      y  y  y  y     win,pb,pix
0x20 24  0  8  8  8  0 16  0  0 0 0x22DC      y  y  y  y     win,pb,pix
0x21 24  0  8  8  8  0 24  0  0 0 0x22DC      y  y  y  y     win,pb,pix
0x22 24  0  8  8  8  0 24  8  0 0 0x22DC      y  y  y  y     win,pb,pix
0x23 24  0  8  8  8  0 32  0  0 0 0x22DC      y  y  y  y     win,pb,pix
0x24 24  0  8  8  8  0  0  0  4 1 0x22DC      y  y  y  y     win,pb,pix
0x25 24  0  8  8  8  0 16  0  4 1 0x22DC      y  y  y  y     win,pb,pix
0x26 24  0  8  8  8  0 24  0  4 1 0x22DC      y  y  y  y     win,pb,pix
0x27 24  0  8  8  8  0 24  8  4 1 0x22DC      y  y  y  y     win,pb,pix
0x28 24  0  8  8  8  0 32  0  4 1 0x22DC      y  y  y  y     win,pb,pix

Device platform:
eglinfo: eglInitialize failed
```

**Comment by meatballs** [comment]
previous version still working fine. I've redacted stuff that looked like api keys.
```
warp-terminal
16:06:29 [INFO] Spawning terminal server process...
16:06:29 [INFO] Running app with windowing system: X11
16:06:29 [INFO] Running terminal server...
16:06:29 [INFO] Crash reporting setting is disabled; not initializing sentry.
16:06:29 [INFO] Starting warp with channel state ChannelState { <redacted> } and version Some("v0.2024.04.16.08.02.stable_00")
16:06:29 [INFO] Performance metrics collector started
16:06:29 [INFO] Flushed telemetry events.
16:06:29 [INFO] Initializing app services
libEGL warning: DRI2: failed to authenticate
16:06:29 [WARN] SQLite error 283 (A WAL mode database file was recovered): recovered 191 frames from WAL file /home/owen/.local/state/warp-terminal/warp.sqlite-wal
16:06:29 [INFO] Connecting to SQLite database
16:06:29 [INFO] Attempting to start websocket connection in CloudObjects::Listener
16:06:29 [INFO] fetching team tester status
16:06:29 [INFO] Checking for update on channel stable_release. Update id is Hz6zL4F
16:06:29 [INFO] Fetching channel versions (without changelogs) from Warp server
16:06:29 [INFO] dispatching global action for root_view:open_from_restored
16:06:29 [INFO] Computing available system fonts
16:06:29 [INFO] Starting shell /usr/bin/zsh
16:06:29 [INFO] Successfully spawned tty with pid: 271868
16:06:29 [INFO] Successfully spawned child zsh process with pid 271868
16:06:29 [INFO] Fetching channel versions and changelogs from Warp server
16:06:29 [INFO] dispatching global action for workspace:save_app
16:06:29 [INFO] dispatching global action for workspace:save_app
16:06:29 [WARN] Tried to update block filter query without active_filter_editor_block_index set
16:06:29 [INFO] dispatching global action for workspace:save_app
16:06:29 [WARN] Tried to render share modal without a model
16:06:29 [WARN] No alert dialog was set for the native modal
16:06:29 [WARN] Tried to render share modal without a model
16:06:29 [WARN] No alert dialog was set for the native modal
16:06:29 [INFO] Guessed window scale factor: 1
16:06:29 [INFO] Loaded 220 font families
16:06:29 [INFO] Received InitShell hook
16:06:30 [INFO] Received channel versions from Warp server: dev: ChannelVersion { version_info: VersionInfo { version: "v0.2024.04.23.08.01.dev_00", update_by: None, soft_cutoff: Some("v0.2023.05.12.08.03.dev_00") }, overrides: [] }; preview: ChannelVersion { version_info: VersionInfo { version: "v0.2024.04.23.08.01.preview_00", update_by: None, soft_cutoff: None }, overrides: [] }; canary: ChannelVersion { version_info: VersionInfo { version: "v0.2022.09.29.08.08.canary_00", update_by: None, soft_cutoff: None }, overrides: [] }; beta: ChannelVersion { version_info: VersionInfo { version: "v0.2024.04.23.08.01.beta_00", update_by: None, soft_cutoff: None }, overrides: [] }; stable: ChannelVersion { version_info: VersionInfo { version: "v0.2024.04.16.08.02.stable_00", update_by: None, soft_cutoff: Some("v0.2024.02.27.08.01.stable_03") }, overrides: [VersionOverride { predicate: TargetOS(Linux), version_info: VersionInfo { version: "v0.2024.04.16.08.02.stable_01", update_by: None, soft_cutoff: None } }] }
16:06:30 [INFO] Received channel versions from Warp server: dev: ChannelVersion { version_info: VersionInfo { version: "v0.2024.04.23.08.01.dev_00", update_by: None, soft_cutoff: Some("v0.2023.05.12.08.03.dev_00") }, overrides: [] }; preview: ChannelVersion { version_info: VersionInfo { version: "v0.2024.04.23.08.01.preview_00", update_by: None, soft_cutoff: None }, overrides: [] }; canary: ChannelVersion { version_info: VersionInfo { version: "v0.2022.09.29.08.08.canary_00", update_by: None, soft_cutoff: None }, overrides: [] }; beta: ChannelVersion { version_info: VersionInfo { version: "v0.2024.04.23.08.01.beta_00", update_by: None, soft_cutoff: None }, overrides: [] }; stable: ChannelVersion { version_info: VersionInfo { version: "v0.2024.04.16.08.02.stable_00", update_by: None, soft_cutoff: Some("v0.2024.02.27.08.01.stable_03") }, overrides: [VersionOverride { predicate: TargetOS(Linux), version_info: VersionInfo { version: "v0.2024.04.16.08.02.stable_01", update_by: None, soft_cutoff: None } }] }
libEGL warning: DRI2: failed to authenticate
16:06:30 [INFO] Enabled wgpu backends: Backends(VULKAN | GL | METAL | DX12 | DX11 | BROWSER_WEBGPU)
16:06:30 [INFO] Available wgpu adapters (in priority order):
16:06:30 [WARN] Deprioritizing Vulkan-backed llvmpipe adapter due to Mesa < 24.0.2 (unsupported)
16:06:30 [INFO] Cpu: llvmpipe (LLVM 15.0.7, 256 bits)
	Backend: Gl
	Driver: Unknown
	Can present: true
	Supported texture format: Some(Rgba8UnormSrgb)
16:06:30 [INFO] Cpu: llvmpipe (LLVM 15.0.7, 256 bits)
	Backend: Vulkan
	Driver: llvmpipe (Mesa 23.2.1-1ubuntu3.1~22.04.2 (LLVM 15.0.7))
	Can present: true
	Supported texture format: Some(Bgra8UnormSrgb)
16:06:30 [INFO] Verifying adapter "llvmpipe (LLVM 15.0.7, 256 bits)" is valid...
16:06:30 [INFO] Using Cpu (llvmpipe (LLVM 15.0.7, 256 bits)) for rendering new window.
16:06:30 [WARN] No alert dialog was set for the native modal
16:06:30 [WARN] Tried to render share modal without a model
16:06:30 [INFO] Finished loading 220 fonts
16:06:30 [INFO] fetching updated cloud objects, including generic string objects
16:06:30 [INFO] fetching teams metadata
16:06:30 [INFO] Websocket is ready. Getting latest set of updated objects from CloudObjects::Listener
16:06:30 [INFO] fetching updated cloud objects, including generic string objects
16:06:30 [INFO] Received CommandFinished hook
16:06:30 [INFO] Block finished with new state DoneWithNoExecution
16:06:30 [INFO] Incrementing stage from WarpInput to ScriptExecution
16:06:30 [INFO] Detected that Warp was installed using Apt
16:06:30 [INFO] Received Precmd hook
16:06:30 [WARN] redraw_frame was called 15 times before the frame was drawn
16:06:30 [INFO] window resized
16:06:30 [INFO] dispatching global action for workspace:save_app
16:06:30 [INFO] dispatching global action for workspace:save_app
16:06:30 [INFO] dispatching global action for workspace:save_app
16:06:30 [INFO] Update ready for channel version VersionInfo { version: "v0.2024.04.16.08.02.stable_01", update_by: None, soft_cutoff: Some("v0.2024.02.27.08.01.stable_03") }
16:06:30 [INFO] active window changed: Some(WindowId(0))
16:06:30 [INFO] dispatching global action for root_view:update_quake_mode_state
16:06:30 [INFO] dispatching global action for workspace:save_app
16:06:30 [WARN] Expected to have session for session ID SessionId(171388478928763), but doesn't exist
16:06:30 [WARN] redraw_frame was called 10 times before the frame was drawn
16:06:30 [INFO] Received Bootstrapped hook
16:06:30 [INFO] Configured typeahead mode as ShellReported
16:06:30 [INFO] Parsed shell version string: Some([Number(5), Number(8), Number(1)])
16:06:30 [INFO] Shell is bootstrapped with session_id SessionId(171388478928763)
16:06:30 [INFO] Loading history from file /home/owen/.zsh_history for shell zsh
16:06:30 [INFO] Received CommandFinished hook
16:06:30 [INFO] Block finished with new state DoneWithNoExecution
16:06:30 [INFO] Incrementing stage from Bootstrapped to PostBootstrapPrecmd
16:06:30 [INFO] Received Precmd hook
16:06:30 [INFO] dispatching global action for workspace:save_app
16:06:30 [WARN] redraw_frame was called 6 times before the frame was drawn
16:06:30 [WARN] redraw_frame was called 4 times before the frame was drawn
16:06:30 [INFO] Received InputBuffer hook
16:06:35 [INFO] active window changed: None
16:06:35 [INFO] dispatching global action for root_view:update_quake_mode_state
16:06:35 [INFO] dispatching global action for workspace:save_app
```

**Comment by special-k-anon** [comment]
Can you link me to a prior version of the RPM install?

**Comment by vorporeal** [comment]
Previous version .deb: https://releases.warp.dev/stable/v0.2024.04.16.08.02.stable_00/warp-terminal_0.2024.04.16.08.02.stable.00_amd64.deb

Previous version .rpm: https://releases.warp.dev/stable/v0.2024.04.16.08.02.stable_00/warp-terminal-v0.2024.04.16.08.02.stable_00-1.x86_64.rpm

Previous version pacman package: https://releases.warp.dev/stable/v0.2024.04.16.08.02.stable_00/warp-terminal-v0.2024.04.16.08.02.stable_00-1-x86_64.pkg.tar.zst

We're planning on pushing a new version this afternoon that should revert the changes that caused the problems in the most recent releases.

Really sorry for the stability issues this past week.  We'll be doing a postmortem internally to avoid such problems in the future.

**Comment by alokedesai** [comment]
Hey everyone, we just pushed a new release that should fix these issues. Would you mind updating to the latest version of Warp to ensure that these issues are no resolved? Thanks!

**Comment by meatballs** [comment]
All fixed for me now

**Comment by alokedesai** [comment]
Closing this out now that we have confirmation that this is fixed. Please file a new issue if you're experiencing issues!

**Comment by b0tm1nd** [comment]
Having the same issue Under Debian 12 after installing the latest version 0.2025.04.09.08.11.stable.02
Downgrading to the version  0.2024.04.16.08.02.stable.00  mentioned in this thread resolved the issue.
Workarounds like WGPU_BACKEND=gl or WARP_ENABLE_WAYLAND=1 did not help.
My hardware:
Warp version: Some("v0.2025.04.09.08.11.stable_02")
uname(1) output: Linux sparky 6.8.8-x64v2-xanmod1 #0~20240428.g5a3fcc1 SMP PREEMPT_DYNAMIC Sun Apr 28 06:12:45 UTC x86_64 GNU/Linux
Package type: PackageManager(Apt { distribution_update_disabled_repository: false })
Windowing system: X11 { is_x_wayland: false }
Window manager name: Openbox
##################################################
# wgpu Adapters
##################################################
IntegratedGpu: Mesa Intel(R) HD Graphics (BYT)
	Backend: Gl
	Driver: ? (4.2 (Core Profile) Mesa 22.3.6)
##################################################

Is it possible to run the latest Warp on my integrated Intel graphics?
